app/views/Login/index.tsx

```diff
@@ -199,13 +287,24 @@ function Login(props: Props) {
                         <Button
                             type="submit"
                             name={undefined}
-                            disabled={pending}
+                            disabled={loginPending}
                         >
                             Login
                         </Button>
                     </div>
                 </form>
             </div>
+            {APP_ENV === 'DEV' && (
```

Comment:
Is this type safe?

Reply:
If you are talking about `DEV` being typesafe from env validation, its not. There isn't any form of validation present at the moment. Should I add them? Was that intentional? If I need to add them, what are the other envs I should support? Similar to community dashboard?